pkg/uconn/analyzer.go

```diff
@@ -64,8 +64,9 @@ func (a *analyzer) start() {
 			// create query
 			query := bson.M{}
 
-			if len(datum.Tuples) > 5 {
-				datum.Tuples = datum.Tuples[:5]
+			tuples := datum.Tuples.Items()
```

Comment:
Note that `.Items()` must be called in order to convert the map into an array. 
https://github.com/activecm/rita/commit/c4b5b3fa83b503af6da624a9727cbf1bfbaaa45a